jina/parsers/helper.py

```diff
@@ -311,10 +311,10 @@ def __call__(self, parser, args, values, option_string=None):
         :param values: the values to add to the parser
         :param option_string: inherited, not used
         """
-        if isinstance(values, list):
-            d = [_port_to_int(port) for port in values]
-        elif isinstance(values, str):
-            d = _port_to_int(values)
+        d = []
+        for value in values:
```

Comment:
this sems risky, this is already used in ports successfulyy

Reply:
I think the logic here is correct